cloud/jenkins/psmdb_operator_gke_alpha.groovy

```diff
@@ -0,0 +1,313 @@
+                        runTest('rs-shard-migration', 'scaling')
+                        ShutdownCluster('scaling')
+                   }
+                }
+                stage('E2E Basic Tests') {
+                    steps {
+                        CreateCluster('basic')
+//                        conditionalRunTest('default-cr', 'basic')
+                        runTest('one-pod', 'basic')
+                        runTest('monitoring-2-0', 'basic')
+                        runTest('arbiter', 'basic')
+                        runTest('service-per-pod', 'basic')
+                        runTest('liveness', 'basic')
+                        runTest('users', 'basic')
+                        runTest('data-sharded', 'basic')
+                        ShutdownCluster('basic')
+                    }
+                }
+                stage('E2E SelfHealing') {
+                    steps {
+                        CreateCluster('selfhealing')
+                        runTest('storage', 'selfhealing')
+                        runTest('self-healing', 'selfhealing')
+                        runTest('operator-self-healing', 'selfhealing')
+                        ShutdownCluster('selfhealing')
+                    }
+                }
+                stage('E2E Backups') {
+                    steps {
+                        CreateCluster('backups')
+                        sleep 60
+                        runTest('upgrade', 'backups')
+                        runTest('upgrade-consistency', 'backups')
+                        runTest('demand-backup', 'backups')
+                        runTest('demand-backup-sharded', 'backups')
+                        runTest('scheduled-backup', 'backups')
+                        runTest('upgrade-sharded', 'backups')
+                        runTest('pitr', 'backups')
+                        runTest('pitr-sharded', 'backups')
+                        ShutdownCluster('backups')
+                    }
+                }
+            }
+        }
+    }
+    post {
+        always {
+            setTestsresults()
+            withCredentials([string(credentialsId: 'GCP_PROJECT_ID', variable: 'GCP_PROJECT'), file(credentialsId: 'gcloud-alpha-key-file', variable: 'CLIENT_SECRET_FILE')]) {
+                sh '''
+                    export CLUSTER_NAME=$(echo jenkins-psmdb-$(git -C source rev-parse --short HEAD) | tr '[:upper:]' '[:lower:]')
+                    source $HOME/google-cloud-sdk/path.bash.inc
+                    gcloud auth activate-service-account alpha-svc-acct@"${GCP_PROJECT}".iam.gserviceaccount.com --key-file=$CLIENT_SECRET_FILE
+                    gcloud config set project $GCP_PROJECT
+                    gcloud alpha container clusters delete --zone us-central1-a $CLUSTER_NAME-basic $CLUSTER_NAME-scaling $CLUSTER_NAME-selfhealing $CLUSTER_NAME-backups | true
+                '''
+            }
+            sh '''
+                sudo docker rmi -f \$(sudo docker images -q) || true
```

Comment:
https://github.com/percona/percona-xtradb-cluster-operator/blob/main/Jenkinsfile#L389 please check it

Reply:
https://github.com/Percona-Lab/jenkins-pipelines/pull/1009/commits/b43c55ab86b9ef473cafc63524894ee8c5b9cebd